README.md

```diff
@@ -744,16 +744,20 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
 
 #### Single-click/drag-n-drop upload
 
+- [BlackUpload](https://github.com/FarisCode511/BlackUpload) - An Anonymus File Uploader Script with Multi-File Upload Support. `MIT` `PHP`
```

Comment:
There is no license in the repository, please remove this until a proprer license is added